UnityProject/Assets/Scripts/Items/Implants/Organs/SlimeCore.cs

```diff
@@ -0,0 +1,222 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Chemistry;
+using Chemistry.Components;
+using HealthV2;
+using HealthV2.Living.PolymorphicSystems.Bodypart;
+using Systems.Character;
+using UI.CharacterCreator;
+using UnityEngine;
+
+[RequireComponent( typeof(ReagentCirculatedComponent))]
+public class SlimeCore : BodyPartFunctionality
+{
+
+
+	public int DropDownIndex = 0;
+
+	private CharacterSheet ForCore =>
+		new CharacterSheet()
+		{
+			Species = "SlimeBall",
```

Comment:
Suggestion, we should have a static class with all the species names so when we need them we grab them from there once and not have to manually type them everytime which can cause typos

Reply:
Well it depends on how want it set up, since the costumions is spring is not in code anywhere

Reply:
none of the customisations, have any Hardcoded Value, Also makes it harder to add a custom species